app/src/main/java/ru/prometeydev/movie/model/BaseRepo.kt

```diff
@@ -65,7 +65,7 @@ abstract class BaseRepo {
         }
     }
 
-    private fun handleException(error: Throwable): Throwable {
+    private fun handleException(error: Throwable?): Throwable {
```

Comment:
How to handle an exception if it is null?

Reply:
I will fix it